### PR TITLE
Rename E Function Params - EntityLoadByPK()

### PR DESCRIPTION
**Pull request by shawnoden**
#898

### CODE REVIEW

File: data/en/entityloadbypk.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-		{"name":"id","description":"ID of the entity to be loaded","required":true,"default":"","type":"string","values":[]},
-		{"name":"unique","description":"Lucee4.5+ Specify whether you expect multiple or just one record\nIf set to `true`, the entity is returned\nIf set to `false` an array is returned to support multiple entities\nIf set to `true` but multiple entities would be returned an error will be thrown","required":false,"default":"","type":"boolean","values":[]}
+		{"name":"entity","description":"Name of the entity to be loaded.","required":true,"default":"","type":"string","values":[]},
+		{"name":"id","description":"ID of the entity to be loaded.","required":true,"default":"","type":"string","values":[]},
+		{"name":"unique","description":"LUCEE ONLY: Specify whether you expect multiple or just one record.\nIf set to `true`, the entity is returned.\nIf set to `false` an array is returned to support multiple entities.\nIf set to `true` but multiple entities would be returned an error will be thrown.","required":false,"default":"","type":"boolean","values":[]}
```
Comment (KamasamaK) [inline data/en/entityloadbypk.json]: `Lucee4.5+` is the proper tag to use.

File: data/en/entityloadbypk.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
 {
 	"name":"entityLoadByPK",
 	"type":"function",
-	"syntax":"entityLoadByPK(entityName, id)",
+	"syntax":"entityLoadByPK(entity, id)",
 	"returns":"any",
 	"related":["entityLoadByExample"],
-	"description":" Loads and returns an array of objects for given primary key.",
+	"description":"Loads and returns an array of objects for given primary key.\bUse this function to avoid specifying the required boolean parameter in `EntityLoad()`.",
```
Comment (KamasamaK) [inline data/en/entityloadbypk.json]: Is the `\b` intentional?

Comment (shawnoden) [reply data/en/entityloadbypk.json]: Supposed to be a new line \n. Apparently my fingers were wider and faster than I intended. 

Fixed.

File: data/en/entityloadbypk.json [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
-		{"name":"id","description":"ID of the entity to be loaded","required":true,"default":"","type":"string","values":[]},
-		{"name":"unique","description":"Lucee4.5+ Specify whether you expect multiple or just one record\nIf set to `true`, the entity is returned\nIf set to `false` an array is returned to support multiple entities\nIf set to `true` but multiple entities would be returned an error will be thrown","required":false,"default":"","type":"boolean","values":[]}
+		{"name":"entity","description":"Name of the entity to be loaded.","required":true,"default":"","type":"string","values":[]},
+		{"name":"id","description":"ID of the entity to be loaded.","required":true,"default":"","type":"string","values":[]},
+		{"name":"unique","description":"LUCEE 4.5+ : Specify whether you expect multiple or just one record.\nIf set to `true`, the entity is returned.\nIf set to `false` an array is returned to support multiple entities.\nIf set to `true` but multiple entities would be returned an error will be thrown.","required":false,"default":"","type":"boolean","values":[]}
```
Comment (KamasamaK) [inline data/en/entityloadbypk.json]: I was basically saying to revert the change because that is the way the tag works. Look at how it's styled already in [CFDocs](https://cfdocs.org/entityloadbypk).

Comment (shawnoden) [reply data/en/entityloadbypk.json]: Gotcha. Fixed properly this time.

**Comment by pfreitag** [comment]
Thanks, merged - was this added in CF9? would be good to add a minimum version number here too.

**Comment by KamasamaK** [comment]
According to the Adobe docs, it was added in CF9.
